ipaclient/install/client.py

```diff
@@ -3529,9 +3527,17 @@ def __init__(self, **kwargs):
             raise RuntimeError(
                 "--server cannot be used without providing --domain")
 
-        if self.force_ntpd and self.no_ntp:
+        if self.force_ntpd:
+            raise RuntimeError(
+                "--force-ntpd has been deprecated")
```

Comment:
Do we want to completely fail in this case or just print a nasty warning (and let it fail in the future when the option is removed)?

Reply:
Warning could do better!

Reply:
Yeah, just print a deprecation warning.